travis.sh

```diff
@@ -50,7 +50,7 @@ then
         echo -n "./console tests:sync-ui-screenshots $TRAVIS_BUILD_NUMBER"
         if [ -n "$PLUGIN_NAME" ]
         then
-            echo -n " --repository=piwik/plugin-$PLUGIN_NAME"
+            echo -n " --repository=matomo-org/plugin-$PLUGIN_NAME"
```

Comment:
I don't think this would work for innocraft plugins.

Reply:
that won't work for any plugin not in matomo-org scope, but guess that never worked before. Will replace it with `$TRAVIS_REPO_SLUG`, guess that should work.